src/query/builder.js

```diff
@@ -751,11 +755,23 @@ assign(Builder.prototype, {
     let amt = parseInt(amount, 10);
     if (isNaN(amt)) amt = 1;
     this._method = 'counter';
-    this._single.counter = {
-      column,
-      amount: amt,
-      symbol: (symbol || '+')
-    };
+    let counter = this._single.counter || {};
+
+    if(!counter[column]) {
+      counter[column] = {
+        amount: amt,
+        symbol: (symbol || '+')
+      };
+    } else {
+      //Counter for column already exists. Add/subtract the current column amount by the new amount
+      if(symbol === '-') {
+        counter[column].amount -= amount;
```

Comment:
I think this might actually be buggy. If first do decrement by 10, we set the counter to `{amount: 10, symbol: '-'}` but if we then do another decrement by 5, the value will get changed to `{amount: 5, symbol: '-'}` while it should in fact be changed to `{amount: 15, symbol: '-`}`.

I think we should drop the `symbol` here and simply store the amount as a signed value where positive values represent increments and negative ones represent decrements.